modules/ztp-image-based-upgrade-installing-lifecycle-agent-using-web-console.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-lifecycle-agent-using-web-console_{context}"]
+= Installing the {lcao} by using the web console
+
+You can use the {product-title} web console to install the {lcao} on both the seed and target cluster.
+
+.Prerequisites
+
+* Install the {rh-rhacm} Operator 2.9.2 or later.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Operators* -> *OperatorHub*.
+. Search for the *{lcao}* from the list of available Operators, and then click *Install*.
+. Keep the default selection of *Installation mode* ["All namespaces on the cluster (default)"] and *Installed Namespace* ("openshift-operators") to ensure that the Operator is installed properly.
```

Comment:
This needs to be updated as this operator is namespaced, and *not* installed in all namespaces. See below image.

![image](https://github.com/openshift/openshift-docs/assets/13664224/63c05c6d-2d27-44df-963e-6b47746e0c78)

maybe to take inspiration from the [PTP operator](https://docs.openshift.com/container-platform/4.12/networking/using-ptp.html#install-ptp-operator-web-console_using-ptp).

Reply:
@leo8a I updated this now with the proper namespaces, PTAL

Reply:
/lgtm 

thanks for updating it